src/main/kotlin/blackjack/domain/Game.kt

```diff
@@ -0,0 +1,29 @@
+package blackjack.domain
+
+class Game(private val _dealer: Dealer, private val _gamerList: List<Gamer>) {
+
+    val dealer: Dealer
+        get() = _dealer
+
+    val gamerList: List<Gamer>
+        get() = _gamerList
+
+    constructor(gamerList: List<Gamer>) : this(Dealer(), gamerList)
```

Comment:
```suggestion
class Game(private val _gamerList: List<Gamer>, private val _dealer: Dealer = Dealer()) {

    val dealer: Dealer
        get() = _dealer

    val gamerList: List<Gamer>
        get() = _gamerList
```

게임 클래스 👍 

default parameter를 사용한다면 constructor를 제거해도 됩니다.